lib/project_types/script/messages/messages.rb

```diff
@@ -113,7 +113,7 @@ module Messages
           graphql_error_cause: "An error was returned: %s.",
           graphql_error_help: "\nReview the error and try again.",
 
-          script_repush_cause: "A script with this UUID already exists (UUID: %s).",
+          script_repush_cause: "A version of this script already exists on the app.",
```

Comment:
Since we no longer need the UUID, you'll have to also update this line: https://github.com/Shopify/shopify-app-cli/blob/6e8521863045721a6714660b6784378fd8e907a8/lib/project_types/script/ui/error_handler.rb#L217 Should be:

```ruby
cause_of_error: ShopifyCli::Context.message("script.error.script_repush_cause"),
```